src/components/StatsBoxGrid.tsx

```diff
@@ -563,7 +542,23 @@ const StatsBoxGrid: React.FC<IProps> = () => {
   ]
   const dir = isLangRightToLeft(intl.locale as Lang) ? "rtl" : "ltr"
   return (
-    <Grid>
+    <Grid
```

Comment:
Not blocking: I would see about exploring a different approach in going between `flex` and `grid` in a separate PR. Maybe just not doing that and sticking with Grid?

And I also like `SimpleGrid` here as a smaller component in this simple use case. Personal preference of course! 😄

Reply:
cc @pettinarip here

Reply:
Yeah, agreed. With the `SimpleGrid` we could just do `<SimpleGrid columns={{ base: 1, lg: 2 }}>`, right?

Reply:
@pettinarip yes, this would work here.

I'll rollout a new PR to update this and any additional refactoring as needed.